content/blog/the-importance-of-ubiquitous-language/index.md

```diff
@@ -0,0 +1,181 @@
+
+The second thing PMs should avoid doing is being authoritative about the ubiquitous language.  In order for the language to be ubiquitous, _everyone_ must agree on the meaning.  The business, product, design, and engineering.  This means that there is room for negotiation on the terms being used.  Every term in the ubiquitous language will have clear meaning to the business, but that does not mean that it will have a 1:1 correlation (every time) with the terms used in the business.  For example the user of the online shopping business may use the word Shopping Cart consistently, however the team may determine for one reason or another to use the term "Cart" instead.  As long as the business clearly understands that "cart" is the same as "shopping cart" with no ambiguity, then it can used in place of "shopping cart" in the ubiquitious language.
+
+When producing their artifacts, generally in the form of user stories, they should use ubiquitous language continuously and consistently.  An examples story may be:
+
+> As a customer, I want to be able to add products to my shopping cart, so that I can continue shopping until I'm ready to check out
+
+Let's now explore how this user story may make its way into the designs for this application.
+
+### Design
+
+As a frequent communicator with the business, being good stewards of the ubiquitous and business languages applies to deign the same way it does to product.  They additionally have a large artifact that they generate and maintain for use by the team, and that is the visual designs and user interviews. 
+
+For both artifacts it is important to stay consistent with the ubiquitous language terminology.  By using this language these two artifacts are uniquely positioned to be one of the best methods to iterate and refine the ubiquitous language.  At the end of the day, the design files produced are consumed by both the business and the engineering team.  They are consumed by the business when validating user flows and UI designs, and they are used by the engineering team when reviewing, prepping for, and ultimately implementing the UI and the associated backend elements.  Because it is consumed by and influenced by both of these groups of people, it provides great opportunity to validate the ubiquitous language as well.  If engineers have questions about the terminology used in the design, they may try to make sense of things they see in the design using different terminology or concepts.  Its important that these terms and concepts are de-conflicted with designs so we don't create a translation layer.  Similarly, as the user interviews are happening, if the users are saying things like "this term doesnt make sense here", or "I would have expected this to be labeled x" it's a sign that the ubiquitous language being used isn't valid, and should be iterated upon.
```

Comment:
Meaty paragraph. Consider making it more concise as this point is important. These artifacts are a rallying point for the team to have important conversations and agree on ubiquitous language (or unambiguous terms).  In turn, these artifacts are the most direct connection to validate your language with users interacting with your product.